### ISSUE TITLE
NaN loss occur when change the patch_size

### ISSUE THREAD

**Issue by htuann2712** [Pointcept/PointTransformerV3#150]
Hi @Gofinge, thanks for the interesting work. I'm trying to fine-tune some small modifications in the config. I found that if I set the patch size for the model stages non-uniformly, like this:
`enc_patch_size = (256, 256, 128, 64, 64), 
dec_patch_size = (64, 64, 128, 256)`
it results in the model loss becoming NaN very early in the training iterations. Do you have any suggestions or insights into why this happens?

Thank you!

**Comment by htuann2712** [comment]
It occur when 'enable_flash=True'

**Comment by Gofinge** [comment]
How about set "enable_amp=False"?

**Comment by htuann2712** [comment]
I also tried both "enable_amp=False" with low learning rate but this phenomena still occurred

**Comment by Gofinge** [comment]
> It occur when 'enable_flash=True'

Oh, maybe caused by Flash Attention. Maybe you can adaptively enable flash attention when the patch size larger than 256. I didn’t feel much difference when set to 64.

**Comment by htuann2712** [comment]
yeah, i tried to disable flash attention and it solved. I guess the main problem come from the flash attn, i will trying with traditional attn. Thanks for your respond.
